core/amber/src/main/scala/edu/uci/ics/texera/workflow/operators/huggingFace/HuggingFaceTextSummaryOpDesc.scala

```diff
@@ -0,0 +1,71 @@
+package edu.uci.ics.texera.workflow.operators.huggingFace
+
+import com.fasterxml.jackson.annotation.{JsonProperty, JsonPropertyDescription}
+import edu.uci.ics.amber.engine.common.workflow.{InputPort, OutputPort}
+import edu.uci.ics.texera.workflow.common.metadata.{OperatorGroupConstants, OperatorInfo}
+import edu.uci.ics.texera.workflow.common.metadata.annotations.AutofillAttributeName
+import edu.uci.ics.texera.workflow.common.operators.PythonOperatorDescriptor
+import edu.uci.ics.texera.workflow.common.tuple.schema.{AttributeType, Schema}
+
+class HuggingFaceTextSummaryOpDesc extends PythonOperatorDescriptor {
+  @JsonProperty(value = "attribute", required = true)
+  @JsonPropertyDescription("column to perform text summary on")
```

Comment:
call it `attribute` instead of `column`.

"the attribute to perform text summarization on"